run-tests.sh

```diff
@@ -8,7 +8,7 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 pydocstyle invenio_rest tests docs && \
-isort -rc -c -df && \
+isort --check-only --diff && \
```

Comment:
I was just eaves dropping...

Question: Doesn't it need the name of the folder? Or by default does all?

I mean:

```suggestion
isort invenio_rest tests --check-only --diff && \
```

Reply:
I am not sure, we don't have this anywhere if I am not mistaken... it is not super important because I guess it will check only Python files.

Reply:
@ntarocco when run as it is now you are getting:
```
./run-tests.sh


                 _                 _
                (_) ___  ___  _ __| |_
                | |/ _/ / _ \/ '__  _/
                | |\__ \/\_\/| |  | |_
                |_|\___/\___/\_/   \_/

      isort your imports, so you don't have to.

                    VERSION 5.1.4


Nothing to do: no files or paths have have been passed in!

Try one of the following:

    `isort .` - sort all Python files, starting from the current directory, recursively.
    `isort . --interactive` - Do the same, but ask before making any changes.
    `isort . --check --diff` - Check to see if imports are correctly sorted within this project.
    `isort --help` - In-depth information about isort's available command-line options.

Visit https://timothycrosley.github.io/isort/ for complete information about how to use isort.
```

In the output, which means it is actually not isorting anything :)

If you add the python packages (`invenio_rest tests`), you actually have errors...

```
./run-tests.sh
ERROR: /Users/ppanero/Workspace/inveniosw/invenio-rest/tests/test_csrf.py Imports are incorrectly sorted and/or formatted.
--- /Users/ppanero/Workspace/inveniosw/invenio-rest/tests/test_csrf.py:before	2020-07-22 17:31:45.026645
+++ /Users/ppanero/Workspace/inveniosw/invenio-rest/tests/test_csrf.py:after	2020-07-22 17:34:03.786624
@@ -177,8 +177,9 @@

 def test_csrf_not_signed_correctly(csrf_app, csrf):
     """Test CSRF malicious attempt with passing malicious cookie and header."""
+    from itsdangerous import TimedJSONWebSignatureSerializer
+
     from invenio_rest.errors import RESTCSRFError
-    from itsdangerous import TimedJSONWebSignatureSerializer

     with csrf_app.test_client() as client:
         # try to pass our own signed cookie and header in an attempt to bypass
```

I think the main problem comes from the fact that they don't have the `recursive` flag anymore... and that message commented on top is not an error (it should IMHO) hehe

Reply:
👍 On it!

Reply:
@ppanero thanks a lot for keeping an eye on this, I have fixed it. I have run `pytest` locally, but I should have run `./run-tests.sh`